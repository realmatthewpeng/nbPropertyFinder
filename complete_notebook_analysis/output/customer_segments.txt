Total number of lines found: 10

display(data.describe())
display(samples - data.mean().round())
display(samples - data.median().round())
samples_percentage['Total'] = samples_percentage.sum(axis=1)
samples_w_total['Total'] = samples_w_total.sum(axis=1)
samples_w_total = samples.copy()
y_all = data[column_to_study]
X_all = new_data.drop(column_to_study, axis=1)
new_data = data.copy()
df_preds = pd.concat([df_diffs, pd.Series(sample_preds, name='PREDICTION')], axis=1)

Incorrect Lines:


Precision = 1.0
Recall = 0.8333333333333334

samples_percentage = samples.loc[:,"Fresh":"Delicatessen"].div(samples_w_total['Total'], axis=0) * 100
samples_percentage
