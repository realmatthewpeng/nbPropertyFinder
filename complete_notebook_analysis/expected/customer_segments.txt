display(data.describe())
display(samples - data.mean().round())
display(samples - data.median().round())
samples_w_total = samples.copy()
samples_w_total['Total'] = samples_w_total.sum(axis=1)
samples_percentage = samples.loc[:,"Fresh":"Delicatessen"].div(samples_w_total['Total'], axis=0) * 100
samples_percentage['Total'] = samples_percentage.sum(axis=1)
samples_percentage
new_data = data.copy()
X_all = new_data.drop(column_to_study, axis=1)
y_all = data[column_to_study]
df_preds = pd.concat([df_diffs, pd.Series(sample_preds, name='PREDICTION')], axis=1)